script.js

```diff
@@ -0,0 +1,123 @@
+    completeStateMessage.style.display = "flex"
+    theForm.style.display = "none"
+})
+continueBtn.addEventListener("click",() =>{
+    theForm.style.display = "flex"
+    completeStateMessage.style.display = "none"
+    confirmBtn.disabled = true
+})
+inputCardName.addEventListener("keyup",() =>{
+    displayCardName.textContent = inputCardName.value
+    nameErrorMessage()
+})
+inputCardNumber.addEventListener("keyup",() =>{
+    displayCardNumber.textContent = inputCardNumber.value
+    validateUserInput()
+    numberErrorMessage()
+})
+inputMonth.addEventListener("keyup",() =>{
+    displayMonth.textContent = inputMonth.value
+    validateUserInput()
+    dateErrorMessage()
+})
+inputYear.addEventListener("keyup",() =>{
+    displayYear.textContent = inputYear.value
+    validateUserInput()
+    dateErrorMessage()
+})
+inputCVC.addEventListener("keyup",() =>{
+    displayCVC.textContent = inputCVC.value
+    validateUserInput()
+    cvcErrorMessage()
+})
+function validateUserInput(){
+    if(inputCardNumber.value.length === 16 && inputMonth.value.length === 2 && inputYear.value.length === 2 && inputCVC.value.length === 3
+        && inputMonth.value > 0 && inputMonth.value < 13 && inputYear.value > 21 && inputYear.value < 27){
+        confirmBtn.disabled = false
+    }
+    else{
+        confirmBtn.disabled = true
+    }
+}
+function nameErrorMessage(){
+    if(inputCardName.value === ""){
+        incorrectName.textContent = "cant be blank"   
+        inputCardName.style.border = "ipx solid red" 
+    }
+    else if(inputCardName.value.length < 6){
+        incorrectName.textContent = "value must be between 6 char to 30 char"
+    }
+    else{
+        incorrectName.textContent = ""
+    }
+}
+function numberErrorMessage(){
+    if(inputCardNumber.value === ""){
+        incorrectNumber.textContent = "cant be blank"
+    }
+    else if(inputCardNumber.value.length < 16){
+        incorrectNumber.textContent = "Card Number must be 16 digits"
+    }
+    else{
+        incorrectNumber.textContent = ""
+    }
+}
+function dateErrorMessage(){
+    if(inputMonth.value === "" && inputYear.value === ""){
+        incorrectDate.textContent = "cant be blank"
+    }
+    else if(inputMonth.value.length < 2 || inputYear.value.length < 2){
+        incorrectDate.textContent = "input the correct date format"
+    }
+    else if(inputMonth.value === 00 || inputMonth.value > 12){
+        incorrectDate.textContent = "month must be between 01 to 12"
+    }
+    else if(inputYear.value < 22 || inputYear.value > 26){
+        incorrectDate.textContent = "Year can only be between 2022 to 2026"
+    }
+    else{
+        incorrectDate.textContent = ""
+    }
+}
+function cvcErrorMessage(){
+    if(inputCVC.value === ""){
+        incorrectCVC.textContent = "cant be blank"
+    }
+    else if(inputCVC.value.length < 3){
+        incorrectCVC.textContent = "CVC must be 3 digits"
+    }
+    else{
+        incorrectCVC.textContent = ""
+    }
+}
```

Comment:
Add a new line to the end of this file. Installing prettier will take care of this for you.